kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/coordinatedcommits/CommitCoordinatorBuilder.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.internal.coordinatedcommits;
+
+import java.util.*;
+
+import io.delta.storage.commit.CommitCoordinatorClient;
+import org.apache.hadoop.conf.Configuration;
+
+/** A builder interface for {@link CommitCoordinatorClient}. */
+public abstract class CommitCoordinatorBuilder {
```

Comment:
why do we need this interface? Can't we reuse the interface in `delta-storage` module?

Reply:
As we discussed, we will move the builder interface to `delta-storage` and reuse it later.